packages/mobile/src/store/offline-downloads/sagas.ts

```diff
@@ -208,15 +209,15 @@ export function* startSync() {
 }
 
 export function* handleSetReachable() {
-  yield* call(setPlayCounterWorker, playCounterWorker)
```

Comment:
These deletions fix the issue because setPlayCounterWorker calls `queue.start()`, so we were starting the queue again after stopping it in a different part of the code.

Reply:
But I'm unsure why this code for the play counter worker was here in the first place? We don't do this blockedPlayCounterWorker thing for the other workers, and I couldn't tell what made it necessary for the play counter worker.